dymos/run_problem.py

```diff
@@ -72,8 +75,8 @@ def modify_problem(problem, restart=None, reset_grid=False):
             load_case(problem, case)
 
 
-def run_problem(problem, refine=False, refine_method='hp', refine_iteration_limit=10, run_driver=True,
-                simulate=False, no_iterate=False):
+def run_problem(problem, refine_method='hp', refine_iteration_limit=0, run_driver=True,
+                simulate=False, recorder_file='dymos_simulation.db', restart=None):
```

Comment:
In the past we've used separate files for saving the "solution" of the optimization and the "simulation" of the resulting control profile.  I don't think recorder_file should be saved here.  FIrst, it's only used by the simulation (and not the solution). In the future we can look at allowing the user to specify custom names for the saved solution and simulation files, but lets just leave them hard coded for now.